Makefile

```diff
@@ -0,0 +1,12 @@
+OCAML_SRCFILES = $(shell git ls-files "*.ml" "*.mli")
+REASON_SRCFILES = $(shell git ls-files "*.re" "*.rei")
```

Comment:
Why do we have formatting of reason sources? We eliminated all reason sources already.

Reply:
There are several Reason codes in the fixtures directory. Well, the interface file doesn't exactly exist yet.

Reply:
That's a good opportunity to convert them to OCaml then.

Reply:
Fixture are for e2e test. We need to ensure the plugin works with Reason syntax